### PR TITLE
Flutter v3 + code refactoring

### PR DESCRIPTION
**Pull request by clangenb**
Changes:
* Bump to flutter v3
* changed naming to adhere to flutter conventions
* split up code into several files

### CODE REVIEW

File: example/ios/Runner/Info.plist [hunk 1]
```diff
@@ -22,12 +24,6 @@
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
-	<key>NSCameraUsageDescription</key>
-	<string>你的操作需要访问相机权限</string>
-	<key>NSMicrophoneUsageDescription</key>
-	<string>你的操作需要访问录像机权限</string>
-	<key>NSPhotoLibraryUsageDescription</key>
-	<string>你的操作需要访问照片库权限</string>
```
Comment (clangenb) [inline example/ios/Runner/Info.plist]: Is it really okay to remove this?

**Review by clangenb — COMMENTED** [review]
Nice, looks very good!

I did not expect you to also do a massive refactoring. :)

File: example/ios/Runner/Info.plist (hunk 1 above)
Comment (Eldar2021) [reply example/ios/Runner/Info.plist]: I’m  sorry I forgot to put it on place after I did generation code. If do this kind of mistake again please let me know. But I will try not to.

**Review by clangenb — COMMENTED** [review]
Nice, looks good to me!

**Pull request merged**